<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTime</class>
 <widget class="QWidget" name="DateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DateTime</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>36</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>current date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="infoFrame">
     <property name="minimumSize">
      <size>
       <width>552</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>960</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="timeClockLable">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>14</y>
        <width>581</width>
        <height>38</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>167000</width>
        <height>167000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="dateWidget" native="true">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>52</y>
        <width>521</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>153</width>
        <height>20</height>
       </size>
      </property>
      <widget class="QLabel" name="dateLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>421</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>153</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>167000</width>
         <height>18000</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="chgzonebtn">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>88</y>
        <width>130</width>
        <height>36</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Change time zone</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chgtimebtn">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>88</y>
        <width>130</width>
        <height>36</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Change time</string>
      </property>
     </widget>
     <zorder>timeClockLable</zorder>
     <zorder>dateWidget</zorder>
     <zorder>chgtimebtn</zorder>
     <zorder>chgzonebtn</zorder>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QFrame" name="syncFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>552</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>960</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="hourFrame">
       <property name="minimumSize">
        <size>
         <width>552</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>960</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>158</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
